README.md

```diff
@@ -69,6 +69,9 @@ Flags:
                                   maps. 0 means no limit.
       --profiling-duration=10s    The agent profiling duration to use. Leave
                                   this empty to use the defaults.
+      --cpu-sampling-frequency=19
```

Comment:
```suggestion
      --profiling-cpu-sampling-frequency=19
```
It'd be good to have a namespace/group. We have too many flags.

related https://github.com/parca-dev/parca-agent/pull/1075